Sofa/framework/Type/src/sofa/type/RGBAColor.h

```diff
@@ -22,33 +22,57 @@
 #pragma once
 #include <sofa/type/config.h>
 
-#include <sofa/type/fixed_array.h>
-
 #include <ostream>
 #include <istream>
 #include <string>
 #include <cmath>
+#include <array>
+#include <algorithm>
+
+#include <sofa/type/fixed_array.h>
```

Comment:
```suggestion
```

Is it still used?

Reply:
Yes, for the compat stuff (fromVec4, ctor)
But I can fwd them and add in the include only in the cpp.